db/seeds.dev.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+FactoryBot.create_list(:account, 3)
+
+Account.find_each do |account|
+  host_ids = SupportedSsg.all.map { |s| [s.os_major_version, s.os_minor_version] }.uniq.flat_map do |version|
+    FactoryBot.create_list(:host, rand(2..3), account: account.account_number, os_version_arr: version).map(&:id)
```

Comment:
Would it make sense to use `ENV` vars or cli arguments for the `rand`, like `MIN_HOSTS`/`MAX_HOSTS`?

Reply:
Some of the stuff yes, but not sure if it's a good idea to have a bunch of ENV vars. It's way easier to just edit the file when you want something custom before running it.

Reply:
It could be useful for QEs to target different amounts via the ENV vars in their test/staging environments, where it could be cumbersome to edit a file.